src/components/modals/CategoryFilterModal.vue

```diff
@@ -164,5 +246,17 @@ export default class CategoryFilterModal extends Vue {
     get unselectedCategories(): string[] {
         return this.$store.getters["modFilters/unselectedCategories"];
     }
+
+    get disabledDates() {
+        return { from: new Date() };
+    }
+
+    get highlightedCreatedDates() {
+        
```

Comment:
How/where is this used?

Also empty line in an odd place.

Reply:
Nowhere, seems like I've forgotten this here 👍